code/controllers/subsystem/retirement.dm

```diff
@@ -0,0 +1,71 @@
+// retirement.dm
+// Copyright 2021 Patrick Meade.
+//
+// This program is free software: you can redistribute it and/or modify
```

Comment:
Any reason you license entire files under AGPL when the repo is already AGPL? Not a complaint, just curious

Reply:
I originally started because it was recommended practice in the license itself:
https://github.com/ScorpioStation/ScorpioStation/blob/master/LICENSE-AGPL3.txt#L627

Later, I realized it's like using a crappy lock to lock up your bike. It won't save your bike from being stolen, but breaking the lock will (beyond a reasonable doubt) establish the thief's intent to steal something.